alica_engine/src/engine/expressionhandler/ExpressionHandler.cpp

```diff
@@ -45,66 +39,47 @@ ExpressionHandler::~ExpressionHandler()
 /**
  * Attaches expressions and constraints to the plans. Called by the AlicaEngine during start up.
  */
-void ExpressionHandler::attachAll()
+void ExpressionHandler::attachAll(PlanRepository& pr, IConditionCreator& cc, IUtilityCreator& uc, IConstraintCreator& crc)
```

Comment:
@hendrik-skubch  Expressionhandler has been reduced to just this function call. Shall we remove the class altogether?

Reply:
Awesome!